dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -95,4 +101,297 @@ class Api::V1::SectionsControllerTest < ActionController::TestCase
 
     assert_equal @course.id, json['courseId']
   end
+
+  test 'logged out cannot create a section' do
+    post :create
+    assert_response :forbidden
+  end
+
+  test 'student cannot create a section' do
+    sign_in @word_user_1
+    post :create
+    assert_response :forbidden
+  end
+
+  test 'teacher can create a section' do
+    sign_in @teacher
+    post :create
+    assert_response :success
+    refute_nil returned_section
+  end
+
+  test 'creating a section returns the section summary' do
+    sign_in @teacher
+    post :create
+
+    # See section_test.rb for tests covering the shape of the section summary.
+    assert_equal returned_section.summarize.with_indifferent_access,
+      returned_json.with_indifferent_access
+  end
+
+  test 'current user owns the created section' do
+    sign_in @teacher
+    post :create
+
+    assert_equal @teacher.name, returned_json['teacherName']
+    assert_equal @teacher, returned_section.user
+  end
+
+  test 'cannot override user_id during creation' do
+    sign_in @teacher
+    post :create, params: {user_id: (@teacher.id + 1)}
+    assert_response :success
+    # TODO: Better to fail here?
```

Comment:
Okay - I'm going to leave this as future cleanup then (strong params, rejecting requests that fail validation instead of going with defaults) except for the `login_type` case which it sounds like we ought to address for the current feature.